operators/src/plot/temporal_raster_mean_plot.rs

```diff
@@ -0,0 +1,438 @@
+use crate::engine::{
+    ExecutionContext, InitializedOperator, InitializedOperatorImpl, InitializedPlotOperator,
+    Operator, PlotOperator, PlotQueryProcessor, PlotResultDescriptor, QueryContext, QueryProcessor,
+    QueryRectangle, RasterQueryProcessor, TypedPlotQueryProcessor,
+};
+use crate::error;
+use crate::util::math::average_floor;
+use crate::util::Result;
+use async_trait::async_trait;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::plots::{AreaLineChart, Plot, PlotData};
+use geoengine_datatypes::primitives::{Measurement, TimeInstance, TimeInterval};
+use geoengine_datatypes::raster::{Pixel, RasterTile2D};
+use serde::{Deserialize, Serialize};
+use snafu::ensure;
+use std::collections::BTreeMap;
+
+pub const TEMPORAL_RASTER_MEAN_PLOT_NAME: &str = "Temporal Area Mean Plot";
+
+/// A plot that shows the mean values of rasters over time as an area plot.
+pub type TemporalRasterMeanPlot = Operator<TemporalRasterMeanPlotParams>;
+
+/// The parameter spec for `TemporalRasterMeanPlot`
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct TemporalRasterMeanPlotParams {
+    /// Where should the x-axis (time) tick be positioned?
+    /// At either time start, time end or in the center.
+    time_position: TemporalRasterMeanPlotTimePosition,
```

Comment:
if we'd want to be correct we'd have at least 2 data points for each raster time slice at beginning and end because in essence we have a stepping function of valid data points. it's a bit dangerous to connect the points like you do because it implies there are values inbetween

Reply:
Well, the user has to know the data. We can't make it correct here for every use case. For the ebv case, for instance, it seemed to be right this way. This way you can see a trend if you like. A bar chart for every time step is another plot type to consider.